src/test/test_utils.cpp

```diff
@@ -273,6 +276,79 @@ void read4x4RgbJpg(size_t& filesize, std::unique_ptr<char[]>& image_bytes) {
     return readImage("/ovms/src/test/binaryutils/rgb4x4.jpg", filesize, image_bytes);
 }
 
+void prepareInferStringRequest(::KFSRequest& request, const std::string& name, const std::vector<std::string>& data, bool putBufferInInputTensorContent) {
+    auto it = request.mutable_inputs()->begin();
+    size_t bufferId = 0;
+    while (it != request.mutable_inputs()->end()) {
+        if (it->name() == name)
+            break;
+        ++it;
+        ++bufferId;
+    }
+    KFSTensorInputProto* tensor;
+    std::string* content = nullptr;
+    if (it != request.mutable_inputs()->end()) {
+        tensor = &*it;
+        if (!putBufferInInputTensorContent) {
+            content = request.mutable_raw_input_contents()->Mutable(bufferId);
+        }
+    } else {
+        tensor = request.add_inputs();
+        if (!putBufferInInputTensorContent) {
+            content = request.add_raw_input_contents();
+        }
+    }
+    tensor->set_name(name);
+    tensor->set_datatype("BYTES");
+    tensor->mutable_shape()->Clear();
+    tensor->add_shape(data.size());
+    size_t dataSize = 1;
+    if (!putBufferInInputTensorContent) {
```

Comment:
Is there test using this function and triggering both if branches for data.size()>1?

Reply:
This is dead code for now since raw format is not supported. Will clean it up, probably putting `FAIL() << "not supported"` there to avoid confusion in the future.

Reply:
If it's dead code make it explicit FAIL.

Reply:
Will be corrected with @michalkulakowski PR with raw input content support for strings